src/style.css

```diff
@@ -28,14 +28,14 @@ body {
   background: #f2f2f2;
 }
 
-/* * {
+* {
```

Comment:
I'd like the scramble and stats to stay selectable. Is this sufficient for you?
```suggestion
.timer {
```

Reply:
I tried this change out. Unfortunately is has some weird, hard to describe behavior. Take a look at the following gif:

1. First I do a regular solve after long pressing. Everything looks great! (notably, the time text didn't get selected).
2. Then, I select some text in the scramble and do a solve. The scramble regenerates, and it looks like nothing is selected again. So so, still great.
3. Then, I do third solve (for this one, I didn't long press before starting the timer, I just did a fairly short tab). Note how some random text in the scramble gets selected? Pretty weird.

![timer-touch-demo](https://user-images.githubusercontent.com/277474/94360402-beed9100-0061-11eb-953a-6289a375226b.gif)


I played around with this a bit. I think the issue is related to the fact that when you turn on `user-select: none` for an element, it actually makes it impossible to deselect text by tapping on the `user-select: none` element. I tried quite a bit of Googling and couldn't find anyone talking about this issue, though, so I'm kind of stumped.

I tried some random stuff, and one possible workaround might be to not use this css rule and instead call `preventDefault` on the timer related `TouchEvent`s. I haven't thought too hard about if there are weird side effects of that, and I haven't tested if it works ok on other mobile browsers and iOS...

Reply:
Per discussion offline:

1. I've updated this PR with your suggestion.
2. As a possible future improvement, we could clear the current selection when starting the timer, which might work around the bug described above.

Reply:
Thanks! I'll dig into what causes #3.